docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,615 @@
+- `-p` - If you want the server to run on a port different than what is specified in the configuration .yaml file, you can override the existing port.
+
+### Enroll bootstrap user with TLS CA
+
+Now that your TLS CA is configured and before you can deploy any other nodes for your organization, you need to enroll the bootstrap (admin) user of the TLS CA. Since the CA server is up and running, instead of using the **Fabric CA server CLI commands** we now use the **Fabric CA client CLI commands** to submit an enrollment request to the server.
+
+Performed by using the Fabric CA client, the enrollment process is used to generate the certificate and private key pair which forms the node identity. You should have already setup the required folders in the [Fabric CA client](#fabric-ca-client) section.
+
+The folder structure that we are using for these Fabric CA client commands is:
+  ```
+  fabric-ca-client
+    └── tls-ca
+    └── tls-root-cert
+  ```
+  These folders are used by the Fabric CA client to:
+   - Store the certificates that are issued when the Fabric CA client enroll command is run against the TLS CA server to enroll the TLS CA bootstrap identity. (**tls-ca** folder)
+   - Know where the TLS CA root certificate resides that allows the Fabric CA client to communicate with the TLS CA server. (**tls-root-cert** folder)
+
+1. Copy the TLS CA root certificate file `fabric-ca-server-tls/ca-cert.pem`, that was generated when the TLS CA server was started, to the `fabric-ca-client/tls-root-cert/tls-ca-cert.pem` folder. Notice the file name is changed to `tls-ca-cert.pem` to make it clear this is the root certificate from the TLS CA. **Important:** This TLS CA root certificate will need to be available on each client system that will run commands against the TLS CA.
+
+2. The Fabric CA Client also needs to know where Fabric CA client binary is located. The `FABRIC_CA_CLIENT_HOME` environment variable is used to set the location.
+
+   ```
+   export FABRIC_CA_CLIENT_HOME=<FULLY-QUALIFIED-PATH-TO-FABRIC-CA-BINARY>
+   ```
+   For example, if you are in the `fabric-ca-client` folder you can use:
+   ```
+   export FABRIC_CA_CLIENT_HOME=$PWD
+   ```
+3. You are ready to use the Fabric CA client CLI to enroll the TLS CA admin user. Run the command:
+   ```
+   ./fabric-ca-client enroll -d -u https://<ADMIN>:<ADMIN-PWD>@<CA-URL>:<PORT> --tls.certfiles <RELATIVE-PATH-TO-TLS-CERT> --enrollment.profile tls --csr.hosts '<CA_HOSTNAME>' --mspdir tls-ca/tlsadmin/msp
+   ```
+   Replace:
+   - `<PORT>` - with the port that the TLS CA is listening on.
+   - `<ADMIN>` - with the TLS CA admin specified on the `init` command.
+   - `<ADMIN-PWD>` - with the TLS CA admin specified on the `init` command.
+   - `<CA-URL>` - with the hostname specified in the `csr` section of the TLS CA configuration .yaml file.
```

Comment:
Nit:
CA-URL and PORT should be next to each other in this list.

Reply:
done.